src/common/SignupBox.jsx

```diff
@@ -24,6 +24,7 @@ class SignupBox extends React.Component {
     handleClick = () => {
         if (this.state.valid) {
             this.props.onSignup(this.state.email);
+            trackEvent("emailEntered", "Store: Email entered in sign up box");
```

Comment:
a signup-event is dispatched from the [api](https://github.com/BA-Group/sizeme-react/blob/f3a2d4995ea3d927c44d80e275bf3c36855524a7/src/api/sizeme-api.js#L187) already, so this is redundant